src/common/connect.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { ConnectedHost } from './ConnectedHost';
+
+export function connect<T>(
+  component: React.ReactType,
+  storesToSubscribe: string[],
+  getProps: (props: T, ...stores) => T
```

Comment:
I'm not quite sure how to do store keys without making this much more verbose. Is this what you're meaning?

Instead of this:

```ts
let stores = { items: new ItemsStore(), selection: new SelectionStore };
```

We'd do this:

```ts
let itemsStoreKey = storeKey<IItemsStore>('items');
let selectionStoreKey = storeKey<ISelectionStore>('selection');

let stores = new StoreCollection();
stores.expose(itemsStoreKey, new ItemsStore());
stores.expose(selectionStoreKey, new SelectionStore());
```